vllm/distributed/utils.py

```diff
@@ -361,6 +361,11 @@ def stateless_destroy_torch_distributed_process_group(
         stateless_init_torch_distributed_process_group().
     """
     # Lazy import for non-CUDA backends.
-    from torch.distributed.distributed_c10d import _shutdown_backend
-    _shutdown_backend(pg)
+    try:
+        # pytorch <= 2.6
+        from torch.distributed.distributed_c10d import _shutdown_backend
+        _shutdown_backend(pg)
+    except ImportError:
+        # pytorch >= 2.7
+        pg.shutdown()
```

Comment:
I used this in the PPLX PR.  I think this is a bit nicer than try/catch.
```
    if is_torch_equal_or_newer("2.7"):
        pg.shutdown()
    else:
        # Lazy import for non-CUDA backends.                                                                                                                            
        from torch.distributed.distributed_c10d import _shutdown_backend
        _shutdown_backend(pg)
```

Reply:
feel free to change it in the moe modularization pr!